template/build/webpack.prod.conf.js

```diff
@@ -44,7 +44,7 @@ module.exports = merge(baseWebpackConfig, {
       minify: {
         removeComments: true,
         collapseWhitespace: true,
-        removeAttributeQuotes: true
+        removeAttributeQuotes: false
```

Comment:
This is actually not related to this pull request, but is necessary for quasar framework to work with the phonegap developer app.